Modules/mmapmodule.c

```diff
@@ -32,9 +32,7 @@
 #  include <unistd.h>             // close()
 #endif
 
-// to support MS_WINDOWS_SYSTEM OpenFileMappingA / CreateFileMappingA
-// need to be replaced with OpenFileMappingW / CreateFileMappingW
-#if !defined(MS_WINDOWS) || defined(MS_WINDOWS_DESKTOP) || defined(MS_WINDOWS_GAMES)
+#if !defined(MS_WINDOWS_SYSTEM)
```

Comment:
@eryksun, @zooba: Is my change here correct? I pulled in main, resolved the conflicts, and did what I assumed the now removed comment meant :)

Reply:
I think the entire ifdef can be removed now - it's excluding the entire implementation for those builds, which shouldn't be necessary with the updated APIs. I _believe_ this module is now supported for all Windows API set variants.

There's also going to be a reference in PC\config.c I assume, since that should have a reference to `init_mmap`.

Reply:
Removed; let's see what the CI says.